src/com/hoffrogge/uebungen/KlassenWeitergeben.java

```diff
@@ -0,0 +1,25 @@
+package com.hoffrogge.uebungen;
+
+import java.util.ArrayList;
+
+import com.hoffrogge.lehreinheit02.Wuerfel;;
+
+public interface KlassenWeitergeben {
+	
+	public ArrayList<Wuerfel> instanceList = new ArrayList<Wuerfel>();
+	
+	/**
+	 * Diese Methode sollte einen W�rfel zu einer Liste von Rechtecksinstanzen hinzuf�gen.
+	 */
+	public void addToInstanceList(Wuerfel wuerfel);
+	
+	/**
+	 * Diese Methode sollte einen W�rfel mit einer Bezeichnung in der Liste finden und zur�ckgeben;
+	 */
+	public Wuerfel getDieByName();
```

Comment:
Naming: getWuerfelNachName (wir wollen in der AG auf Deutsch programmieren);
Parameter fehlt;